src/integral.jl

```diff
@@ -3,7 +3,7 @@
 ################################################################################
 
 """
-    integral(f, geometry[, rule]; FP=Float64)
+    integral(f, geometry[, rule]; dt=FiniteDifference(), FP=Float64)
```

Comment:
What does `dt` stand for? Can this maybe be named a bit more self-explanatory? When I read `dt` I immediately think of a time step size for a numerical solver of an ODE (could also just be my background), but I think generally something like `diff_method` or something along those lines would be easier to understand what this does.

Reply:
I mostly just wanted to get something implemented for testing and figured we could debate the name. I was thinking in terms of like $\frac{\text{d}}{\text{d}t}~\text{geometry}(t)$, but that’s a fair point.

Reply:
I just changed `dt` to `diff_method`

Reply:
Thanks!